config/locales/mfa/zh.yml

```diff
@@ -3,7 +3,7 @@ zh:
   mfa:
     account_info: 添加另外一个身份证实方法可以预防你在失去其他方法后被锁在账户外。
     add: 添加另外一个方法
-    info: 选择一个多因素身份证实方法来增加更多一层安全防线。我们建议你选择至少两个不同方法，以防你丢失其中一个。
+    info: 通过选择多因素身份证实方法来为您的 %{app_name} 帐户添加额外一层安全保障。
```

Comment:
I think there's another exception we can remove now that we're interpolating `%{app_name}` consistently:

https://github.com/18F/identity-idp/blob/c8106fc4e43548929839b5200fd02c62adf5be11/spec/i18n_spec.rb#L22